README.md

```diff
@@ -92,6 +92,8 @@ The SEO tag will respect any of the following if included in your site's `_confi
     ```
 
 * `google_site_verification` for verifying ownership via Google webmaster tools
+* `bing_site_verification` for verifying ownership via Bing webmaster tools
+* `yandex_site_verification` for verifying ownership via Yandex webmaster tools
```

Comment:
We'll need to retain backwards compatibility with a `default` for google, but what do you think about a design pattern like:

```yml
site_verifications:
  google: XX-1234-abc
  bing: asdfasfdasf
  yandex: 12341234
```